recipes/robotframework-interpreter/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "robotframework-interpreter" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 66323c3cc577800a3df08acd0ff642567fcd3f028ad995f816186824b7b305c6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - robotframework>=3.2
+
+about:
+  home: https://github.com/martinRenou/{{ name }}
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
```

Comment:
The license file isn't in the source distribution, so you will need to either manually package it (https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually), or include it upstream (see https://github.com/martinRenou/robotframework-interpreter/pull/1) and put out a new release.